libcaf_core/caf/fwd.hpp

```diff
@@ -269,8 +269,10 @@ using int_gauge_family = metric_family_impl<int_gauge>;
 namespace detail {
 
 class actor_system_access;
+class actor_system_config_access;
 class mailbox_factory;
 class monotonic_buffer_resource;
+class opaque;
```

Comment:
Where is this type defined? Could it be you refactored it?

Reply:
Yeah, that's a leftover from a previous experiment. Good catch!